pkg/asset/manifests/topologies.go

```diff
@@ -34,6 +35,11 @@ func determineTopologies(installConfig *types.InstallConfig) (controlPlaneTopolo
 		infrastructureTopology = configv1.HighlyAvailableTopologyMode
 	}
 
+	if ptr.Deref(installConfig.ControlPlane.Replicas, 0) == 2 &&
+		(installConfig.Arbiter == nil || ptr.Deref(installConfig.Arbiter.Replicas, 0) == 0) {
+		infrastructureTopology = configv1.HighlyAvailableTopologyMode
+		controlPlaneTopology = configv1.DualReplicaTopologyMode
+	}
```

Comment:
nit: This looks like it could be done in above block to set `controlPlaneTopology`. 

https://github.com/openshift/installer/blob/ebe8abe386338ea666827189e5d82b55019de86c/pkg/asset/manifests/topologies.go#L11-L15

Later on this func, if `installConfig.Arbiter != nil`, it will override it. That said, we could add `ptr.Deref(installConfig.Arbiter.Replicas, 0) > 0` to that check on line 33.

Reply:
Hey @qJkee, thanks for the updates. I have one tiny https://github.com/openshift/installer/pull/9534#discussion_r1987769981 below.